modules/navier_stokes/doc/content/source/physics/WCNSLinearFVTwoPhaseMixturePhysics.md

```diff
@@ -0,0 +1,60 @@
+# Navier Stokes Two Phase Mixture using a Linear Finite Volume discretization / WCNSLinearFVTwoPhaseMixturePhysics
+
+!syntax description /Physics/NavierStokes/TwoPhaseMixtureSegregated/WCNSLinearFVTwoPhaseMixturePhysics
+
+## Equation(s)
+
+This [Physics](Physics/index.md) adds terms to the flow and energy equations to account for the presence of a
+two-phase mixture. If specified with the [!param](/Physics/NavierStokes/TwoPhaseMixtureSegregated/WCNSLinearFVTwoPhaseMixturePhysics/add_phase_transport_equation)
+parameter, it can also solve for the advection-diffusion equation of a moving phase fraction.
+
+!alert note
+If the other phase is solid, the [!param](/Physics/NavierStokes/TwoPhaseMixtureSegregated/WCNSLinearFVTwoPhaseMixturePhysics/add_phase_transport_equation)
+parameter should be set to false.
+
+The phase advection-diffusion equation is:
+
+!equation
```

Comment:
This equation doesn't seem right. We don't have a advection-diffusion equation for phase variable in mixture model. Do you get it from other documentation?

Reply:
I ll double check the equation but I think we do solve for the other phase advection in the two phase mixture model

Reply:
We have a advection equation for phase volume/void fraction, but no diffusion. I assume here you mean `INSFVScalarFieldAdvection`, right?

Reply:
This is different from the scalar field used in mixture model. I think it may use for other models. But this equation definitely doesn't show up in the governing equations of mixture model for sure.

Reply:
I only saw this parameter from `lid-driven-two-phase-physics.i`, not `lid-driven-two-phase.i`.  I probably miss something here?

Reply:
`lid-driven-two-phase.i` does not use the Physics syntax

but it does have diffusion
```
  [phase_2_time]
    type = FVFunctorTimeKernel
    variable = phase_2
  []
  [phase_2_advection]
    type = INSFVScalarFieldAdvection
    variable = phase_2
    u_slip = 'vel_slip_x'
    v_slip = 'vel_slip_y'
  []
  [phase_2_diffusion]
    type = FVDiffusion
    variable = phase_2
    coeff = 1e-3
  []
```

Reply:
> This is different from the scalar field used in mixture model.

this could very well be true. I am writing the Physics to replicate what was done with the kernel, I am not starting from the equations and the literature

Reply:
> `lid-driven-two-phase.i` does not use the Physics syntax
> 
> but it does have diffusion
> 
> ```
>   [phase_2_time]
>     type = FVFunctorTimeKernel
>     variable = phase_2
>   []
>   [phase_2_advection]
>     type = INSFVScalarFieldAdvection
>     variable = phase_2
>     u_slip = 'vel_slip_x'
>     v_slip = 'vel_slip_y'
>   []
>   [phase_2_diffusion]
>     type = FVDiffusion
>     variable = phase_2
>     coeff = 1e-3
>   []
> ```

I didn't notice this before. I'll do some search about it. Maybe you are right

Reply:
Now I remember, `lid-driven-two-phase.i` doesn't follow mixture model formulation. It only uses `slip_velocity`, which is a more simple assumption than mixture model. It doesn't have the `drift_flux` kernel in the momentum equation. Both are valid but we should make the documentation more clear on this. I'll do a PR for that